config/locales/en.yml

```diff
@@ -13,8 +13,21 @@ en:
         db_intellectual_property: Intellectual property in DB
         has_non_public_domain_authority: Has non public domain authority
   ingestibles:
```

Comment:
This is controller-related i18n resources, so to use shorthand version e.g. `.success` instead of `ingestible.update.success` we should prefix with controller name (ingestible**s** instead of ingestible)